File: Java/Vetores/soma_vetor.java
// Exercício: Soma do vetor
// Objetivo: armazenar valores em um vetor, calcular a soma e a média dos elementos.

import java.util.Locale;
import java.util.Scanner;

public class soma_vetor {
    public static void main(String[] args) {
        
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int n;
        double media, soma;

        System.out.print("Quantos numeros voce vai digitar? ");
        n = sc.nextInt();

        double[] vet = new double[n];
        for (int i = 0; i < n; i++) {
            System.out.print("Digite um numero: ");
            vet[i] = sc.nextDouble();
        }

        System.out.println();

        System.out.print("VALORES = ");
        for (int i = 0; i < n; i++) {
            System.out.print(String.format("%.1f ", vet[i]));
        }

        soma = 0;
        for (int i = 0; i < n; i++) {
            soma = soma + vet[i];
        }

        media = 0;
        media = soma / n;

        System.out.println();
        System.out.println("SOMA = " + String.format("%.2f", soma));
        System.out.println("MEDIA = " + String.format("%.2f", media));

        sc.close();
    }
}

File: Python/vetores/soma_vetor.py
# Exercício: Soma do vetor
# Objetivo: armazenar valores em uma lista, calcular a soma e a média dos elementos.

n = int(input("Quantos numeros voce vai digitar: "))
vet = [0.0 for x in range (n)]

for i in range (n):
    vet[i] = float(input("Digite um numero: "))

print("\nVALORES = ", end=" ")
for i in range (n):
    print(f"{vet[i]:.1f}", end=" ")    

soma = 0
for i in range (n):
    soma = soma + vet[i]

media = 0
media = soma / n

print(f"\nSOMA = {soma:.2f}" )
print(f"MEDIA = {media:.2f}")